.gitignore

```diff
@@ -1 +1,2 @@
 _output/
+.vscode
```

Comment:
You should use a global gitignore on your system rather then adding specifics to each project. Please remove from this PR.
Here's a guide on how to ignore things system wide: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Oh yea, I spaced on that one. Thanks!